peer.go

```diff
@@ -441,6 +440,7 @@ func newChanMsgStream(f *fundingManager, h *htlcswitch.Switch, p *peer,
 
 // Start starts the chanMsgStream.
 func (c *chanMsgStream) Start() {
+	c.wg.Add(1)
```

Comment:
Yeah, if we stick to that pattern always goroutines should be fairly easy to track :)